### PR TITLE
add section on making requests

### PR DESCRIPTION
**Pull request by braddialpad**
Just adding a small section to the docs on making requests.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -28,15 +28,25 @@ Types of contributions:
...
 - **Contributor:** Has the ability to create PRs and merge their change into `staging` after at least one approving review.
 - **Maintainer:** A trusted contributor with the ability to release Dialtone.
 - **Admin:** Has the ability to change any configuration on the Dialtone repository and release Dialtone. Usually for members of the Dialtone team.
 
-## Making a Pull Request
+## How to Contribute
+
+### Feature Request
+
+If you would like to request a new feature for Dialtone, please fill out and submit [this form.](https://forms.monday.com/forms/8a9a6ff69d7e9f95caee029c2806e2c1?r=use1)
```
Comment (braddialpad) [inline .github/CONTRIBUTING.md:43]: @dialpad-drew not sure if this is live yet, will change if not.

Comment (dialpad-drew) [reply .github/CONTRIBUTING.md:43]: It's live.

**Review by dialpad-drew — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (dbecherdialpad) [reply .github/CONTRIBUTING.md:43]: is this linked anywhere other than a bit of text in this readme?

Comment (braddialpad) [reply .github/CONTRIBUTING.md:43]: Not yet but I'm going to put it in #dialtone

Comment (dialpad-drew) [reply .github/CONTRIBUTING.md:43]: @dbecherdialpad, we also have an [undefined story](https://switchcomm.atlassian.net/browse/DT-324) for us to add info like this to more places so it's not just mentioned once in a repository's markdown file. To wit, the documentation site and storybook should both provide users with _just enough_ information so they know what to do or where to go if they want to request a new feature or report a bug. The challenge for us to solve is to figure out how much information that is and minimize duplicating or triplicating a bunch of documentation. TBD.

I'm also wondering if we think about a CTA in the dialpad.design nav header and/or a callout for reporting a bug or requesting a new feature on the home page.

Comment (braddialpad) [reply .github/CONTRIBUTING.md:43]: > I'm also wondering if we think about a CTA in the dialpad.design nav header and/or a callout for reporting a bug or requesting a new feature on the home page.

I think that's a pretty good idea.

**Pull request merged**